templates/RELEASE_NOTES_unified.htm.tmpl

```diff
@@ -32,7 +32,7 @@
     <MadCap:listOfProxy style="mc-list-of-tag: h3;mc-list-of-class: list;mc-list-of-paragraph-class: RN;" />
 
     {{- range .Components }}
-
+    {{- $length := len .Changelogs }}{{ if ne $length 0 }}
```

Comment:
These need to be split out too but you could maybe just use one statement (if this works: `{{ if ne (len .Changelogs) 0 }}`